### ISSUE TITLE
TST: Windows test failed with Microsoft Visual C++ 14.0 or greater is required

### ISSUE THREAD

**Issue by pllim** [astropy/pytest-doctestplus#177]
Just started happening. Not sure why.

Example log: https://github.com/astropy/pytest-doctestplus/runs/5327364400?check_suite_focus=true

```
_____________ ERROR collecting build/lib.win-amd64-3.8/module2.py _____________
build\lib.win-amd64-3.8\module2.py:1: in <module>
    from _module2 import foo
E   ModuleNotFoundError: No module named '_module2'
!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
============================== 1 error in 0.21s ===============================
---------------------------- Captured stderr call -----------------------------
error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
```

Why are we building C-ext in the test suite here, @lpsinger ?

https://github.com/astropy/pytest-doctestplus/blob/feea9835dd345d2efc9907a4be9f92b0fbe37289/tests/test_doctestplus.py#L951

**Comment by lpsinger** [comment]
> Why are we building C-ext in the test suite here, @lpsinger ?

To test discovering a doctest in a Numpy ufunc.

**Comment by lpsinger** [comment]
I'm quite surprised that the Windows virtual environment for GitHub Actions does not provide a C/C++ compiler. I guess we could either 1) skip this test if there is no compiler or 2) figure out how to install a suitable compiler.

**Comment by pllim** [comment]
But the test was passing before, right? I am so confused.

**Comment by lpsinger** [comment]
That's true. I don't understand either. What if you retry the CI pipeline?

**Comment by pllim** [comment]
OK re-running https://github.com/astropy/pytest-doctestplus/actions/runs/1896158465 🤞

**Comment by pllim** [comment]
Still failed. The Python versions (3.7 and 3.8) should not matter, right?

**Comment by pllim** [comment]
Maybe we're missing the `CC` env? actions/virtual-environments#2503

I don't have time to test this theory today. If you want to open an exploratory PR, that will be great. If not, I'll revisit next week. Thanks!

**Comment by lpsinger** [comment]
This will be fixed by #178.
